recipe/meta.yaml

```diff
@@ -36,12 +36,10 @@ requirements:
     - setuptools_scm >=6.2
     - tomli
     - py-cpuinfo
-    - numpy
+    - numpy >=2
   run:
     - python
-    - numpy >=1.7
-    # Optional dependecy included for ease of use
-    - msgpack-python
```

Comment:
Is there a reason this is being dropped?

Reply:
I dropped this to be consistent with not including other optional dependencies. Happy to put it back and include other optional dependencies here too, but I thought best to mirror the PyPI behaviour which is not to install optional dependencies by default.

Reply:
Given we have been shipping this as a dependency here for so long, think it would be more consistent to leave it

Plus users can get confused about how to install MsgPack as the naming differs

* PyPI: [`msgpack`]( https://pypi.org/project/msgpack )
* Conda: [`msgpack-python`]( https://github.com/conda-forge/msgpack-python-feedstock )

Reply:
👍  - shall I add other optional dependencies where possible?

Reply:
The others feel relatively niche. Some (like `zfpy`) have additional constraints (like NumPy 1 or limitations on version used as is the case with `pcodec`). So think we could leave them out and let users install them if they like